test/e2e/run_e2e.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# PURPOSE: This script runs our pytest integration test suite.
+#
+# PRELIMINARY:
+#  You must have a functioning MIE deployment. Set the required environment variables; see the testing readme for more
+#  details.
+#
+# USAGE:
+#  ./run_integ_tests.sh $component
```

Comment:
Where is the $component parameter used in this script?

Reply:
It's not used in the script. That section is describing the usage of how to invoke the script. This is outlined in the general testing readme:

"These tests are invoked by running the run_integ.sh script in the test/integ directory. The script takes a positional argument for what component to run the tests on:

./run_integ.sh workflowapi
./run_integ.sh dataplaneapi"

I do notice a typo in the file name that I will change.